docs/xamarin/users-android.mdx

```diff
@@ -99,7 +99,7 @@ You should call Helpshift SDK's login API whenever the user successfully logs in
   </tr>
 </table>
 
-```csharp
```

Comment:
csharp is correct. Applies to all code in this file.

Reply:
Ack